docs/user-guide/dual-mesh.ipynb

```diff
@@ -0,0 +1,353 @@
+{
```

Comment:
<p>Can you use the <code>.relabel()</code> method with a legend for this instead?</p><p> </p><p>Below is an example: </p><p> </p><p><a href="https://uxarray--859.org.readthedocs.build/en/859/user-guide/topological-aggregations.html#:~:text=%22edge%22)-,(%0A%20%20%20%20uxds.uxgrid.plot(line_color%3D%22black%22,500%2C%20title%3D%22Node%20to%20Edge%20Aggregation%20(Mean)%22),-Edge%20Aggregations" target="_blank">https://uxarray--859.org.readthedocs.build/en/859/user-guide/topological-aggregations.html#:~:text=%22edge%22)-,(%0A%20%20%20%20uxds.uxgrid.plot(line_color%3D%22black%22,500%2C%20title%3D%22Node%20to%20Edge%20Aggregation%20(Mean)%22),-Edge%20Aggregations</a> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/859/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p><code>.relabel()</code> doesn't seem to work when combining plots of lines. It works in your notebook because everything you combined are points. Do you know how to make it work for actual grid plots? @philipc2</p>